omnij-core/src/main/java/foundation/omni/tx/EncodeMultisig.java

```diff
@@ -21,7 +21,7 @@
 
     private static final int maxKeys = 3;  /* Redeemable key + 2 data keys */
     private static final int maxDataKeys = maxKeys - 1;
-    private static final int dustAmountInSatoshis = 100000;
+    private static final int dustAmountInSatoshis = 684;
```

Comment:
This commit c4d5071c05a451843a74034bb354996795bb4587 should be the correct solution.